Dockerfile

```diff
@@ -109,6 +109,7 @@ COPY patches ./patches
 COPY src ./src
 COPY webpack ./webpack
 COPY .env.oss \
+  .env.example \
```

Comment:
~Why does the container depend on this file now?~ _I see how the smoke tests uses it._